waspc/cli/src/Wasp/Cli/Command/Common.hs

```diff
@@ -34,3 +41,22 @@ throwIfExeIsNotAvailable exeName explanationMsg = do
     Nothing ->
       E.throwError $
         CommandError ("Couldn't find `" <> exeName <> "` executable") explanationMsg
+
+deleteDirectoryIfExists :: Path' Abs (Dir d) -> Command ()
+deleteDirectoryIfExists dir = do
+  cliSendMessageC $ Msg.Start $ "Deleting the " ++ dirName ++ " directory..."
+  dirExist <- liftIO $ IOUtil.doesDirectoryExist dir
+  if dirExist
+    then deleteDir
+    else -- todo(filip): do we need to report this? If not, we can simply call the function from IOUtil.
```

Comment:
This is a question for the reviewers as well :)

Reply:
Both is fine really, whatever you like better, I think it doesn't matter much.

I think output I would like to see is something like
```
Cleaning up Wasp project...
- deleted ...
- deleted ...
Done!
```

since this is quite informative, but if there are no dirs/files to be deleted, I would expect it to be just
```
Cleaning up Wasp project...
Done!
```

Reply:
Ok, let's leave it as is.